docs/ Finch API/8-Testing.md

```diff
@@ -35,6 +41,27 @@ This scenario provides a much larger dataset to test against. Some attributes of
 * multiple departments and work locations, and
 * payments with various earning types, deductions, and employer contributions.
 
+### Examples
+
+Payments  endpoint for `smallco`:
```

Comment:
This won't necessarily be for `smallco` since it's bound by the auth token. So perhaps:

"An example call to the payments endpoint:"

Reply:
The alternative is that we give them a runnable example (e.g. like I did in the thread). It doesn't hurt for us to share this key - then it would be explicitly for `smallco`.